terraform/examples/alicloud-ots-table/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "alicloud" {
+}
+
+resource "alicloud_ots_table" "table" {
+  provider = "alicloud"
+  table_name = "${var.table_name}"
+  primary_key {
+    name = "${var.primary_key_1_name}"
+    type = "${var.primary_key_integer_type}"
+  }
+  primary_key {
+    name = "${var.primary_key_2_name}"
+    type = "${var.primary_key_integer_type}"
+  }
+  primary_key {
+    name = "${var.primary_key_3_name}"
+    type = "${var.primary_key_integer_type}"
+  }
+  primary_key {
+    name = "${var.primary_key_4_name}"
+    type = "${var.primary_key_string_type}"
+  }
```

Comment:
If primary_key can be a list, it will be better like:
```
primary_key  = [
  {
    name = "${var.primary_key_1_name}"
    type = "${var.primary_key_integer_type}"
 },
{
    name = "${var.primary_key_2_name}"
    type = "${var.primary_key_integer_type}"
 }
...
]
```

Reply:
Done, thanks.